SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -157,6 +163,26 @@ SpatialGDK::QueryConstraint UGridBasedLBStrategy::GetWorkerInterestQueryConstrai
 	return Constraint;
 }
 
+FVector2D UGridBasedLBStrategy::GetActorLoadBalancingPosition(const AActor& Actor) const
+{
+	USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(Actor.GetNetDriver());
+	const Worker_EntityId ActorEntityId = NetDriver->PackageMap->GetEntityIdFromObject(&Actor);
+
+	if (ActorEntityId == SpatialConstants::INVALID_ENTITY_ID)
+	{
+		return FVector2D(SpatialGDK::GetActorSpatialPosition(&Actor));
+	}
+
+	const SpatialGDK::ComponentData* PositionComponentPtr =
+		NetDriver->Connection->GetCoordinator().GetView()[ActorEntityId].Components.FindByPredicate(
```

Comment:
This change breaks `GIVEN_moving_actor_WHEN_actor_crosses_boundary_THEN_should_relinquish_authority` as it expects the GridLBStrat to instantly change its decision on `ShouldHaveAuthority`, and in reality this is delayed.

Reply:
Not an issue after removing Spatial Position change.